rsocket-core/src/main/java/io/rsocket/Invalidate.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.rsocket;
+
+interface Invalidate {
+
+  void expire();
```

Comment:
on caches the terminology is typically `invalidate()`, so I'd go with that. Maybe rename the class to `Invalidatable` for drawing a parallel with `Disposable`/`Closeable`/etc... (even though that's not a word 😆)?

Reply:
fixed